src/main/resources/sparql/ask-ssn.sparql

```diff
@@ -0,0 +1,14 @@
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX persoon: <https://data.vlaanderen.be/ns/persoon#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+
+ASK WHERE {
+  GRAPH <${graph}> {
+    ?gsIdentificator <https://data.vlaanderen.be/ns/generiek#lokaleIdentificator> ?lokaleIdentificator.
+    ?registrationId <https://data.vlaanderen.be/ns/generiek#gestructureerdeIdentificator> ?gsIdentificator.
+    ?person <https://data.vlaanderen.be/ns/persoon#registratie> ?registrationId.
+     FILTER(?lokaleIdentificator = "${ssn}" && CONTAINS(STR(?person),"personen") && ?person != <http://data.lblod.info/id/personen/${personId}>).
```

Comment:
It can be risky to assume the format of a URI (contains `personen` for example) because it can change over time. If you can find a way to use the type instead, it's a more stable information !

Reply:
updated